src/en/proposals/revolution-rework.md

```diff
@@ -0,0 +1,166 @@
+It's also worth considering revolution as a major mode rather than a minor mode.
+
+### Concept Abstracts
+- unique scoring system
+- voluntary entry into the revolution
+- rev icon replacement
+- additional avenues for recruitment
+- rev uplink 
+- various rebel paraphernalia
+- cult-like alert
+
+## Background
+**You can read a more comfortable and beautiful version here --->** <https://docs.google.com/document/d/1KaitkDoAZFSBrNIc78K522gtLkDHlZaMzH9vNMe9Oz4/edit?usp=sharing>
+
+It's contains additional comments, illustatrations, images and other MAYBE useful information.
+
+**RUS VERSION --->** <https://docs.google.com/document/d/11Lpip1kL9xyVAVoQn-PVuqBHv-stovz8ZaweRDGwxtE/edit?usp=sharing>
+
+## 1. GENERAL MECHANICS
+### 1.1 Rev Uplink
+- Headrevs have another distinctive feature, the rev **uplink implant**. It contains a variety of often decrepit, rusty and makeshift equipment.
+- For **one recruited** it gives **1 Rebel Telecrystal** (RTC) to headrev’s uplink.
+> A full list of gear is being worked on.
+Approximately:
+Soviet-made pistols, Mosinky, PPSh, Kalashnikovs, RGDs, various body armor, clothing, riot posters, homemade war makings, ammunition and other paraphernalia
+
+**FOR WHAT:** These things will make the headrev stand out from the crowd or help him in some minor actions for later climax (like raiding an security patrol).
+
+### 1.2 Holographic Armband
+- In exchange for the icons, the revolutionary will be given the **ability "make intentions known"** which will create a red or blue-red **armband** on his shoulder for the headrevs.
+- The **ability is one-time** and there is no way to remove this bandage once activated.
+- To know the player's affiliation before activation requires examining the character - it will answer it in examine window (being a revolutionary).
+- Once alerted _(1.6)_, all armbands automatically appear on the shoulders of the rebels.
+
+**FOR WHAT:** To get rid of stealthy revolutions, add "rebel entourage" and replace trivial meta icons.
+
+### 1.3 Mindshield
+Has no effect on revolutionaries, but prevents further recruitment _(1.8)_.
+
+### 1.4 Goldstein Books 
+- There is a **free** item in headrev uplink _(1.1)_ - a **“brainwash book”.**
+- Initially there is no books in the uplink _(1.1)_, but for every **[headrev count * 2]** revolutionary, the copy will be replenished by one piece. **[Maximum books = 6]**.
+- When read **(by pressing Z)**, the character becomes a revolutionary. If a loyalist or rebel does it, nothing happens.
+>Hereinafter, a loyalist is a character with a mindshield implant. In the points system, a loyalist and a head are different people.
+
+**FOR WHAT:** Diversify the method of recruitment, allow simple revolutionaries to recruit, and make a reference to 1984.
+
+### 1.5 Rebel Flag
+- There is another free item in headrev uplink _(1.1)_ - rebel flag, which **can be placed** in different departments.
```

Comment:
I have some issues with this, how do you determine if a flag is placed in a department? We don't use department zones by design (it's against one of our pillars of "everything being modifyable by players"), since that creates artificial areas that can't be changed. 

As an example: What happens if medical gets bombed and they decide to turn botany into the new medbay? Does the system think that the flag is placed in botany when it's really placed in med? What happens when multiple departments end up sharing the same space? (this might happen on some maps)

Reply:
As far as I know there is already a system in ninja mechanics where you have to plant a bomb in a department and detonate it. The same mechanics will be used to check the flag location on the station.